### PR TITLE
Theme the hover outlines

### PR DESCRIPTION
**Pull request by jasmussen**
This PR uses the admin color scheme to theme the outline hover style.

Screenshots:

screen shot 2018-06-04 at 08 53 01
screen shot 2018-06-04 at 08 52 48
screen shot 2018-06-04 at 08 53 11
screen shot 2018-06-04 at 08 53 19
screen shot 2018-06-04 at 08 54 59
screen shot 2018-06-04 at 08 55 08
screen shot 2018-06-04 at 08 55 19
screen shot 2018-06-04 at 08 56 40

### CODE REVIEW

**Review by karmatosed — APPROVED** [review]
Great little touch.

File: webpack.config.js [hunk 1]
```diff
@@ -48,49 +48,57 @@ const extractConfig = {
...
 								toggle: '#a3b9a2',
 								button: '#a3b9a2',
+								outlines: '#5E7D5E',
 							},
 							'admin-color-sunrise': {
 								primary: '#d1864a',
 								secondary: '#c8b03c',
 								toggle: '#c8b03c',
 								button: '#d1864a',
+								outlines: '#AA652C',
```
Comment (noisysocks) [inline webpack.config.js]: nit: We should lowercase these strings for consistency 🙂

**Review by noisysocks — APPROVED** [review]
Cool change. Code looks good!

File: webpack.config.js [hunk 2]
```diff
@@ -48,49 +48,57 @@ const extractConfig = {
 							secondary: '#11a0d2',
 							toggle: '#11a0d2',
 							button: '#0085ba',
+							outlines: '#007cba',
 						},
 						themes: {
```
Comment (tofumatt) [inline webpack.config.js]: Unrelated to this PR, but it seems weird we put themes in the webpack config. Any context for why we do that or should I make an issue to move 'em out?

Comment (youknowriad) [reply webpack.config.js]: We put theme here to automatically generate the required CSS files using a post-css plugin #6739

Comment (youknowriad) [reply webpack.config.js]: We can already use `postcss.config.js` and move all the config their. I personally don't like the proliferation of root level config files, but no strong opinion.

**Pull request merged**